cmd/serve.go

```diff
@@ -90,6 +97,17 @@ func serverRunE(cmd *cobra.Command, args []string) error {
 
 	v := viper.GetViper()
 
+	auditLogPath := v.GetString("audit.log.path")
+	fd, err := helpers.OpenAuditLogFileUntilSuccessWithContext(ctx, auditLogPath)
+	if err != nil {
+		return fmt.Errorf("failed to open audit log file: %w", err)
+	}
+	// The file descriptor shall be closed only if the gin server is shut down
+	defer fd.Close()
+
+	// Set up middleware with the file descriptor
+	mdw := ginaudit.NewJSONMiddleware("my-test-component", fd)
```

Comment:
is `my-test-component` what we actually want here?

Reply:
lmao, I thought I added this to a test. I'll fix it.